iwyu.cc

```diff
@@ -2548,12 +2548,14 @@ class IwyuBaseAstVisitor : public BaseAstVisitor<Derived> {
 
```

Comment:
Commit message:  "when there is a explicit instantiation definition visible" -> "... *an* explicit..."